Makefile

```diff
@@ -5,7 +5,7 @@ ALL:
 	npm i
 
 check:
-	npm t
+	-find test/*.js test/standalone/*.js | xargs -n 1 node | node_modules/tap-spec/bin/cmd.js
```

Comment:
yep. this way travis displays a mocha style spec. 

there might be a better alternative for travis. 

we could run tap-spec as the default `npm test` and have something like `make nyan` for the preferred nyan-cat terminal output (unfortunately not helpful/intended for travis ci output).